runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -933,12 +933,11 @@ void KernelGenerator::visit(const ir::operation::StridedSlice &node)
   auto begin_mask = node.param().begin_mask;
   auto end_mask = node.param().end_mask;
   auto shrink_axis_mask = node.param().shrink_axis_mask;
-  auto rank = node.param().rank;
 
   auto fn = std::make_unique<ops::StridedSliceLayer>();
 
   fn->configure(input_alloc, starts_alloc, ends_alloc, strides_alloc, output_alloc, begin_mask,
-                end_mask, shrink_axis_mask, rank);
+                end_mask, shrink_axis_mask);
```

Comment:
Maybe, you should change getting static rank to get dynamic rank

Reply:
@ragmani https://github.com/Samsung/ONE/issues/2338#issuecomment-645084710 happens for static inference. I think dynamic case is also same - they need to use inferred shape, not the first given shape. Could you please explain more why two cases should be changed differently?

Reply:
You are right. I haven't seen it properly.